onnxscript/ir/passes/common/clear_metadata_and_docstring.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""Clear all metadata and docstring from the model, graphs, nodes, and functions."""
+
+from __future__ import annotations
+
+__all__ = [
+    "ClearMetadataAndDocStringPass",
+]
+
+import logging
+
+from onnxscript import ir
+
+logger = logging.getLogger(__name__)
+
+
+class ClearMetadataAndDocStringPass(ir.passes.InPlacePass):
```

Comment:
Would this be called in _optimizer.py (by default)?

Reply:
I don't think it should. This pass should only be used when we ship models. By default the optimizer produces a debugable (but optimized) model for users, which contains all the traces and metadata.